README.rst

```diff
@@ -18,15 +18,34 @@ Installation
 Usage
 =====
 
-::
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Running an instance of Axelrod's second tournament:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> players = [axlf.Player(name) for name in axlf.characteristics.keys()
+   >>>            if axlf.characteristics[name]["original_rank"] is not None]
```

Comment:
I'm not sure I agree. `all_strategies` is the list of all the strategies in `TourExec` (some that were are were apparently not in the tournament). We could create another list for `tournament_strategies` perhaps?